dyn_em/module_initialize_real.F

```diff
@@ -4437,6 +4439,22 @@ SUBROUTINE init_domain_rk ( grid &
 
       ENDIF
 
+      if (config_flags%madwrf_cldinit == 1)  &
+          call Init_madwrf_clouds (moist, p_qv, p_qc, p_qi, p_qs, p00, grid%t_2, grid%p_hyd, grid%ph_2, grid%phb,      &
```

Comment:
@pedro-jm 
Similar to line number 4450 (where f_qc, f_qi, an f_qs) are checked, should those tests be implemented here?

Reply:
I see the test in phy_init